docs/services/qix-engine/introduction.md

```diff
@@ -30,7 +30,50 @@ API | Description
 [QIX](./apis/qix/introduction.md) | Document consumption using JSONRPC and WebSocket protocols.
 [REST](./apis/rest/qlik-associative-engine-api.md) | REST API.
 [Data Connector](./apis/data-loading/introduction.md) | High-performance data loading using the gRPC protocol.
-[Analytical Connector](./apis/server-side-extension/analytical-connector-api.md) | Server side extensions using the gRPC protocol.
+[Analytical Connector](./apis/server-side-extension/introduction.md) | Server side extensions using the gRPC protocol.
+
+## Command Line Parameters
+
+The Qlik Associative Engine has a number of command line parameters that can be used to toggle functionality.
+These parameters are passed when starting a Qlik Associative Engine docker container:
+
+Using `docker run`:
+
+`docker run qlikcore/engine:<version> -S <setting1>=<value> -S <setting2>=<value> ...`
```

Comment:
I think brackets [] is more often brackets used when describing an optional parameter?
https://help.qlik.com/en-US/sense/June2018/Subsystems/Hub/Content/Scripting/Backus-Naur-formalism.htm

Reply:
All command line parameters are optional if that is what you mean? Or are you referring to `...`?